### DISCUSSION TITLE
Put IRequestHandler and INotificationHandler into MediatR.Contracts

### DISCUSSION THREAD

**Discussion by jpa67** [LuckyPennySoftware/MediatR#1008] [Ideas]
Hi,

if we had IRequestHandler and INotificationHandler in MediatR.Contracts,
we could define a handler in a .net standard library.

That could help a lot in isolating a domain layer in a DDD/Clean/Onion architecture.

best Regards

**Comment by BazInTheAZ** [comment]
This has been discussed previously here: #693

**Reply by jpa67** [reply]
Yes but I'm not agree with the reason to not doing it ;)

Then for now I will use my own version with this modification...
